### ISSUE TITLE
Mention Flatpak as installation method

### ISSUE THREAD

**Issue by Cliveptr** [bitcoin-core/bitcoincore.org#758]
This issue tracker is for issues with the Bitcoin Core project website at
[bitcoincore.org](https://bitcoincore.org).

[bitcoin.org](https://bitcoin.org) is not connected with or maintained by the
Bitcoin Core project contributors. Issues with that website should be reported
at [the bitcoin.org
repository](https://github.com/bitcoin-dot-org/bitcoin.org/issues).

General bitcoin questions and/or support requests are best directed to the
Bitcoin StackExchange at https://bitcoin.stackexchange.com.

Title pretty much says it all. See: https://bitcoincore.org/en/download/

**Comment by katesalazar** [comment]
Flatpak is not mentioned in the linked page. NAK.

**Comment by MarcoFalke** [comment]
Makes sense to add the flatpak, given that the snap is listed. Pull requests welcome, I guess?
